### PR TITLE
[Windcentrale] Add units to measurements using QuantityType and fixes

### PR DESCRIPTION
**Pull request by wborn**
* Add units to kwh, kwhForecast, powerAbsTot, powerAbsWd, powerRel, runPercentage and runTime channels
* Update and improve documentation
* Fix partially implemented runtime channel
* Fix wrong millId of De Boerenzwaluw

### CODE REVIEW

File: addons/binding/org.openhab.binding.windcentrale/ESH-INF/thing/millThing.xml [hunk 1]
```diff
@@ -57,49 +58,55 @@
 		<item-type>Number</item-type>
 		<label>Wind speed</label>
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.windcentrale/ESH-INF/thing/millThing.xml]: Does the also deserve a dimension or is there none

Comment (cweitkamp) [reply addons/binding/org.openhab.binding.windcentrale/ESH-INF/thing/millThing.xml]: We can use the quantity javax.measure.quantity.Speed for it. But I don't now the unit Beaufort is available.

Comment (wborn) [reply addons/binding/org.openhab.binding.windcentrale/ESH-INF/thing/millThing.xml]: Currently there is no Beaufort unit in SmartHomeUnits. Beaufort is also a scale and not a unit. 
Should Beaufort also be added to the SmartHomeUnits @htreu, @kaikreuzer?
If so, what would be the dimension? Speed, WindSpeed (new) or Dimensionless?

**Comment by martinvw** [comment]
@marcelrv can you do a quick review?

File: addons/binding/org.openhab.binding.windcentrale/ESH-INF/thing/millThing.xml (hunk 1 above)
Comment (htreu) [reply addons/binding/org.openhab.binding.windcentrale/ESH-INF/thing/millThing.xml]: The mathematical formula seems to be a sort of one-way function from a speed in km/h or kn to Bft. When having 5Bft one can not ensure a correct conversion to a real speed.
That being said I don't see a valid conversion from Bft into any other unit, so I suggest to not add it to the SmartHomeUnits.

**Pull request merged**